src/app/app.config.ts

```diff
@@ -0,0 +1,8 @@
+import { ApplicationConfig } from '@angular/core';
+import { provideRouter } from '@angular/router';
```

Comment:
suggestion

          import { provideRouter } from '@angular/router';

          Explantion: The import statement is correct, ensure that there are adequate tests for the router configuration you are providing.